examples/variables.tf

```diff
@@ -35,11 +35,13 @@ variable "user_id" {
 variable "compartment_id" {
   description = "compartment id where to create all resources"
   type        = string
+  default     = null
```

Comment:
The value set here is relative to the behavior explained on the next review comment.

Having `null` value for `compartment_id` allows the user to execute the example code with minimal edit: just the oci provider authentication variables.

If the user copy/paste his existing `terraform.tfvars` file where `compartment_id` value is set, this will effectively overwrite this default value. Seems to me it is best of both worlds.

Reply:
No, please don't this as null. Please request the user to explicitly pass their compartment id.

Reply:
OK, I guess we can add information on the example usage listing which variables **must** be set by the user before using the example.

That's basically anything related to auth + compartment.